### ISSUE TITLE
false positive, unused variable

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#3173]
```abap
constants lc_sel_fieldname type string value 'SELECTED'.
field-symbols <ls_selection> type voided.
field-symbols <lr_sel_field> type voided.
assign component lc_sel_fieldname of structure <ls_selection> to <lr_sel_field>.
```

![image](https://github.com/abaplint/abaplint/assets/5888506/e8db02d5-c821-4aff-a7d7-5025cc12b434)
